packages/desktop-ui-lib/src/lib/settings/plugins/component/plugin-marketplace/plugin-marketplace-item/dialog-app-selector/dialog-app-selector.component.html

```
@@ -0,0 +1,61 @@
+	<nb-card-header class="mb-0">
+		<div class="header-content">
+			<div class="header-left">
+				<div class="icon-container">
+					<nb-icon icon="external-link-outline"></nb-icon>
+				</div>
+				<div class="header-info">
+					<h5 class="title m-0">{{ 'PLUGIN.DIALOG.APP_SELECTOR.TITLE' | translate }}</h5>
+					<p class="subtitle m-0">{{ 'PLUGIN.DIALOG.APP_SELECTOR.SUBTITLE' | translate }}</p>
+				</div>
+			</div>
+			<button nbButton ghost size="small" (click)="dismiss()" class="close-button">
```

HTMLHint (1.9.2)

[warning] The type attribute must be present on <button> elements.

(button-type-require)